source/tyr/readme.md

```diff
@@ -714,13 +714,27 @@ Finally if you want to delete a poi_type you just have to use the DELETE action:
 
 #### Migrate from POI to OSM
 
-Remove all POI datasets stored in database.
+Load POI from OSM.
 
-    DELETE $HOST/v0/instances/$INSTANCE_NAME/actions/migrate_from_poi_to_osm
+    PUT $HOST/v0/instances/$INSTANCE_NAME/actions/migrate_from_poi_to_osm
```

Comment:
We should add that it must be followed by a delet_dataset/poi (otherwise we can have a problem on reload, no?)
In that case we may even add it to the json response.

Reply:
Also a comment on how adding a poi file switches automatically to poi (and so why this command is useful) would be nice.